lib/fpm/package/pacman.rb

```diff
@@ -0,0 +1,394 @@
+# -*- coding: utf-8 -*-
+require "fpm/package"
+require "backports"
+require "fileutils"
+require "find"
+
+class FPM::Package::Pacman < FPM::Package
+
+  opt_depends = []
+  option "--optdepends", "PACKAGE",
```

Comment:
For multivalue options, this trickery isn't needed anymore. Clamp has an option `:multivalued => true` that does this for us which is kind of nice.

Example:

```ruby
option "--optdepends", "PACKAGE", "Add an optional dependency to the pacman package", :multivalued => true
```

You don't necessarily need to change this, but just a heads up that clamp supports this well now :)